dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/datacite/ResourceDTO.java

```diff
@@ -0,0 +1,487 @@
+package edu.harvard.iq.dataverse.export.datacite;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlText;
+
+import java.util.List;
+
+@JacksonXmlRootElement(localName = "resource")
+@JsonPropertyOrder({ "xsiSchemaLocation", "xmlns", "xmlnsXsi", "identifier", "creators", "titles", "publisher",
+        "publicationYear", "resourceType", "relatedIdentifiers", "descriptions", "contributors", "fundingReferences" })
+public class ResourceDTO {
```

Comment:
I feel that we are overusing a little `*DTO` postfix. Maybe we could rename it to something more associated with the resulting format like `DataCiteResource`? When we naming it `*DTO` I strongly feel that it is used somwhere on the UI or it is exposed over api.